charts/signoz/tests/signoz-instance_env_test.yaml

```
@@ -137,3 +137,88 @@
+            value: tcp://sample.clickhouse.com:9000/?username=$(CLICKHOUSE_USER)&password=$(CLICKHOUSE_PASSWORD)
+
+# --- Test Case 8: External ClickHouse creds with secret and dsn environment variable ---
+  - it: should set external clickhouse creds with secret and dsn environment variable
+    set:
+      clickhouse:
+        enabled: false
+      externalClickhouse:
+        host: "sample.clickhouse.com"
+        cluster: "cluster"
+        user: "user"
+        password: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)